excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/endpoint/exception/SpreadsheetValidationErrorMessageCreatorTest.java

```diff
@@ -416,18 +416,18 @@ public void caseTypeEntityInvalidUserRole_createErrorMessageCalled_customMessage
             .thenReturn(
                 definitionDataItem(SheetName.AUTHORISATION_CASE_TYPE,
                     new ImmutablePair<>(ColumnName.CASE_TYPE_ID, "case type"),
-                    new ImmutablePair<>(ColumnName.USER_ROLE, "user role"))
+                    new ImmutablePair<>(ColumnName.ACCESS_PROFILE, "access profile"))
             );
 
         assertEquals(
-            "Invalid idam role 'user role' in AuthorisationCaseType tab for case type 'case type'",
+            "Invalid IDAM access profile 'access profile' in AuthorisationCaseType tab for case type 'case type'",
             classUnderTest.createErrorMessage(error));
     }
 
     @Test
     public void caseTypeEntityInvalidUserRole_createErrorMessageCalled_customMessageReturned_whenCaseRoleMissing() {
```

Comment:
REname test when validation error is renamed

Reply:
Fixed in 3951e2b